samples/bootloader/README.rst

```diff
@@ -247,35 +256,35 @@ To reduce the development time and potential issues with this route, let the exi
 
 For building and running this sample with an application, see :ref:`ug_bootloader_adding_immutable`.
 
-Building and running using |SES|
+Building and running using |VSC|
 ================================
 
 .. include:: /includes/build_and_run_bootloader.txt
 
 To add the bootloader sample as a child image to your application, complete the following steps:
 
 1. :ref:`Create a private key in PEM format <ug_fw_update_keys>`.
-#. To enable the |NSIB|, run ``menuconfig`` on your application:
+#. Enable the |NSIB| through Kconfig as follows:
 
-   a. Select :guilabel:`Project` > :guilabel:`Configure nRF Connect SDK project`.
-   #. Go to :guilabel:`Modules` > :guilabel:`Nordic nRF Connect` > :guilabel:`Bootloader` and set :guilabel:`Use Secure Bootloader` to enable :kconfig:option:`CONFIG_SECURE_BOOT`.
-   #. Under :guilabel:`Private key PEM file` (:kconfig:option:`CONFIG_SB_SIGNING_KEY_FILE`), enter the path to the private key that you created.
+   a. Select :guilabel:`Kconfig` in the :guilabel:`ACTIONS` panel to open the Kconfig configuration tab.
+   #. Expand :guilabel:`Modules` > :guilabel:`nrf` > :guilabel:`Nordic nRF Connect` > :guilabel:`Bootloader` and set :guilabel:`Use Secure Bootloader` to enable :kconfig:option:`CONFIG_SECURE_BOOT`.
+   #. Expand :guilabel:`Use Secure Bootloader` and enter, under :guilabel:`Private key PEM file` (:kconfig:option:`CONFIG_SB_SIGNING_KEY_FILE`), the path to the private key that you created.
       If you choose to run the sample with default debug keys, you can skip this step.
 
       You can also modify other additional configuration options, but that is not recommended.
       The default settings are suitable for most use cases.
 
       .. note::
          If you need more flexibility with signing, or if you do not want the build system to handle your private key, choose :kconfig:option:`CONFIG_SB_SIGNING_CUSTOM`, and also specify :kconfig:option:`CONFIG_SB_SIGNING_COMMAND` and :kconfig:option:`CONFIG_SB_SIGNING_PUBLIC_KEY`.
+         You can use the :guilabel:`Search modules` bar in nRF Kconfig to find these options.
          These options allow you to define the signing command.
 
-   #. Click :guilabel:`Configure`.
-
-#. Select :guilabel:`Build` -> :guilabel:`Build Solution` to compile your application.
+   #. Click :guilabel:`Save`.
 
+#. Select :guilabel:`Build` in the :guilabel:`ACTIONS` panel to start the build process.
    The build process creates two images, one for the bootloader and one for the application, and merges them.
 
-#. Select :guilabel:`Build` -> :guilabel:`Build and Run` to program the resulting image to your device.
+#. Select :guilabel:`Flash` in the :guilabel:`ACTIONS` panel to program the resulting image to your device.
```

Comment:
Maybe you could use one of the images from Zigbee QSG? See the image under step 4 in the Step 2 section. https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_zigbee_qsg.html

Reply:
@greg-fer Are they even following the new style guide for screenshots?